Adapt demo-qt4 to new repository structure

jordiandreu opened the pull request:
Adapt to new repository structure and use the bin/mxcube
as default launcher for the example application.

Modifies the checkout of the mxcube repository to the local
container.
Modifies the Dockerfile and script to run the mxcube using
the bin/mxcube launcher as ENTRYPOINT.

IvarsKarpics commented on dev/Debian9_qt4/manager_mxcube2_start:
```diff
@@ -0,0 +1,19 @@
+    docker exec -it $CONTAINER /bin/bash
+else
+    echo "Reusing the container $CONTAINER"
+    MXCUBE_ROOT=/home/jandreu/Development/MXCuBE/mxcube
```
Is there a way to avoid hardcoded paths?

jordiandreu committed:
Adapt demo/qt4 to new repository structure

Adapt demo/qt4 to new repository structure and use the bin/mxcube
as default launcher for the example application.

Modifies the checkout of the mxcube repository to the local
container.
Modifies the Dockerfile and script to run the mxcube using
the bin/mxcube launcher as ENTRYPOINT.

jordiandreu commented:
Sorry @IvarsKarpics , I committed the wrong files. These are the files for the demo qt4 master container.
